src/components/common/Extensions/TitledContent/TitledContent.tsx

```diff
@@ -1,12 +1,24 @@
 import React, { FC, PropsWithChildren } from 'react';
+import { useIntl } from 'react-intl';
 import { TitledContentProps } from './types';
+import { useMobile } from '~hooks';
 
 const displayName = 'common.Extensions.TitledContent';
 
-const TitledContent: FC<PropsWithChildren<TitledContentProps>> = ({ children, title, className }) => {
+const TitledContent: FC<PropsWithChildren<TitledContentProps>> = ({
+  children,
+  title,
+  className,
+  isTitleHiddenOnDesktop,
+}) => {
+  const isMobile = useMobile();
+  const { formatMessage } = useIntl();
+
   return (
     <div className={className}>
-      <h4 className="uppercase text-xs text-gray-400 font-medium font-inter mb-2">{title}</h4>
+      {isTitleHiddenOnDesktop && isMobile && (
+        <h4 className="uppercase text-xs text-gray-400 font-medium font-inter mb-2">{formatMessage(title)}</h4>
```

Comment:
You use `{formatMessage(title)}` or `{formatMessage({id: title})}`
What is the difference? Why do you use it in two ways?

Reply:
I'm using `{formatMessage(title)}` when title is passed as prop, and `{formatMessage({id: title})}` when getting string from en.json